topics/admin/metadata.yaml

```diff
@@ -15,7 +15,7 @@ material:
     name: "dev-to-production"
     zenodo_link: ""
     galaxy_tour: ""
-    hands_on: "github"
```

Comment:
Can you let with github and add "enable: false"? This tutorial is not ready to be online